### PR TITLE
feat(skies): Add Sky/Sun Matrix and View from Viewport

### PR DESCRIPTION
**Pull request by chriswmackey**
This commit adds components to create sun and sky matrices.  It also adds a component to create a View from a Rhino Viewport and a component to create a SensorGrid from the floors of rooms.

Most of the code used to capture the properties of the rhino viewport can be found in [this module of the ladybug_rhino library](https://github.com/ladybug-tools/ladybug-rhino/blob/master/ladybug_rhino/viewport.py).

One thing I wanted to check with you @mostapharoudsari is whether I was correct to leave out a hoys input on the sky matrix and sun matrix components (the previous HB[+] has these components). I saw that the latest classes don't have a property for hoys and so I was wondering where in the workflow you were thinking uses could filter the Wea by datetimes.

### CODE REVIEW

File: honeybee_grasshopper_radiance/src/HB Sensor Grid from Rooms.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+-
+
+    Args:
+        _rooms: A list of honeybee Rooms for which sensor grids will be generated.
+        _grid_size: Number for the size of the grid cells.
+        _dist_floor_: Number for the distance to move points from the floors of
+            the input rooms. The default is 0.8 meters.
+    
+    Returns:
+        grid: An SensorGrid object that can be used in a grid-based recipe.
```
Comment (mostaphaRoudsari) [inline honeybee_grasshopper_radiance/src/HB Sensor Grid from Rooms.py:24]:
```suggestion
        grid: A SensorGrid object that can be used in a grid-based recipe.
```

File: honeybee_grasshopper_radiance/src/HB Sensor Grid from Rooms.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Generate a SensorGrid objects from the floors of honeybee Rooms.
```
Comment (mostaphaRoudsari) [inline honeybee_grasshopper_radiance/src/HB Sensor Grid from Rooms.py:11]:
```suggestion
Generate SensorGrid objects from the floors of honeybee Rooms.
```

File: honeybee_grasshopper_radiance/src/HB Sky Matrix.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
```
Comment (mostaphaRoudsari) [inline honeybee_grasshopper_radiance/src/HB Sky Matrix.py:1]: This is the old header. The new one doesn't have my name.

Comment (chriswmackey) [reply honeybee_grasshopper_radiance/src/HB Sky Matrix.py:1]: Good catch.

**Comment by chriswmackey** [comment]
Thank you for the review, Mostapha. We are good to merge, then.

**Pull request merged**